demo/setup.py

```diff
@@ -14,7 +14,7 @@ def read_relative_file(filename):
 README = read_relative_file('README')
 VERSION = '0.1'
 PACKAGES = ['demoproject']
-REQUIRES = ['django-generic-filters', 'mock',
+REQUIRES = ['django-generic-filters', 'mock', 'Django<1.7'
```

Comment:
Should we just put django there and test different versions of Django in the tox matrix?